### DISCUSSION TITLE
Questioning if this is a bug or not not finding the chd or bios files

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#20] [General]
![image](https://github.com/sam-ludlow/mame-ao/assets/7123294/b597f9fe-6ac7-4420-b454-0fbf544f2404)

**Comment by sam-ludlow** [comment]
Hello @paulmortlock 

This machine needs a laserdisc that hasn't been dumped yet, so it won't work.

https://mame.spludlow.co.uk/Machine.aspx?name=maddog2

Cheers
Sam

![image](https://github.com/sam-ludlow/mame-ao/assets/47115020/77170bf0-be88-4f39-9943-4ec50066d43e)

**Comment by paulmortlock** [comment]
aha
